jenkins/node-inspect.jenkinsfile

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent any
```

Comment:
If you do `agent('jenkins-workspace')` does it run on machines with that label? I'm pretty sure that's what we're doing in CI right now.

cc/ @rvagg

Reply:
Yes, using `agent { label 'jenkins-workspace' }` will force execution onto agents with the specified label. From what I can tell in [the docs](https://jenkins.io/doc/book/pipeline/syntax/#agent), the `node(label)` shorthand doesn't seem to exist for `agent`.